suite/cstest/include/test_detail_bpf.h

```diff
@@ -20,6 +20,8 @@ typedef struct {
 	char *ext;
 	char *mem_base;
 	uint32_t mem_disp;
+	bool is_pkt;
+	bool is_signed;
```

Comment:
```suggestion
	tbool is_pkt;
	tbool is_signed;
```

Because in C `false == 0` we could distinguish between an unset `is_pkt` or false. This is why we have to do it as I described before.